_posts/en/newsletters/2020-01-29-newsletter.md

```diff
@@ -0,0 +1,215 @@
+`SIGHASH_ANYPREVOUTANYSCRIPT` signature hash that may simplify
+management of payments in eltoo-based payment channels.  Also included
+are our regular sections for popular Bitcoin StackExchange Q&A and
+notable changes to popular Bitcoin infrastructure and documentation
+projects.
+
+## Action items
+
+- **Upgrade to LND 0.9.0-beta:** this new major version [release][lnd
+  0.9.0-beta] brings improvements to the access control list mechanism
+  ("macaroons"), support for receiving [multipath payments][topic
+  multipath payments], the ability to send additional data in an
+  encrypted onion message (see [Newsletter #81][news81 lnd3900]), native
+  rebalancing support (see [Newsletter #74][news74 lnd3739]), support
+  for requesting channel close payouts go to a specified address (e.g.
+  your hardware wallet; see [Newsletter #76][news76 lnd3655]), and many
+  other features and bug fixes.
+
+- **Help test Bitcoin Core 0.19.1rc1:** this upcoming maintenance
+  [release][bitcoin core 0.19.1] includes several bug fixes.
+  Experienced users are encouraged to help test for any regressions or
+  other unexpected behavior.
+
+## News
+
+- **Breaking the link between UTXOs and unannounced channels:** Bastien
+  Teinturier [posted][teinturier post] to the Lightning-Dev mailing list about changing
+  what data is added to a [BOLT11][] invoice for payments sent to
+  unannounced channels---channels that aren't advertised to the LN
+  network and which usually don't route payments for other users.  The
+  proposed change would remove information from an invoice that could be
+  used to identify the channel's deposit UTXO, replacing it with a
+  one-time per-invoice keypair and a secret that's routed as part of the
+  onion-encrypted payment.  This would require special support from both the
+  spender and the peer who's able to route to the unannounced channel,
+  but other nodes along the routing path would not need to change their
+  implementation.  Teinturier requested feedback on the proposal,
+  including any suggestions for how to eliminate the need to include an
+  encrypted secret in the payment.
+
+- **Layered commitments with eltoo:** Anthony Towns [described][towns
+  layered commitments] a potential modification to his previous
+  [anyprevout proposal][bip-anyprevout] (a variation of
+  [SIGHASH_NOINPUT][topic sighash_noinput]) that could simplify
+  [eltoo][topic eltoo]-based LN channels.  As currently proposed,
+  eltoo-based LN implementations would need to ensure they don't accept
+  a payment whose refund timeout occurs before the payment's
+  unilateral-close delay, otherwise a paying node will be able to
+  reclaim its payment before the receiving node has a chance to use
+  their adaptor signature ("point lock") to legitimately claim the
+  payment.  This differs from current-style LN payments where the
+  timeout and the delay may be chosen independently.
+
+    To allow eltoo to achieve a similar independence of timeout and
```

Comment:
a similar -> similar

Reply:
"a similar" feels better to me, but happy to change it if you feel strongly about it.